ProcessesApi/V1/Helpers/ProcessHelper.cs

```diff
@@ -0,0 +1,36 @@
+using ProcessesApi.V1.Services.Exceptions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ProcessesApi.V1.Helpers
+{
+    public static class ProcessHelper
+    {
+        public static void ValidateFormData(Dictionary<string, object> requestFormData, List<string> expectedFormDataKeys)
+        {
+            expectedFormDataKeys.ForEach(x =>
+            {
+                if (!requestFormData.ContainsKey(x))
+                    throw new FormDataNotFoundException(requestFormData.Keys.ToList(), expectedFormDataKeys);
+            });
+        }
+
+        public static void ValidateOptionalFormData(Dictionary<string, object> requestFormData, List<string> expectedFormDataKeys)
+        {
+            var atLeastOneKey = expectedFormDataKeys.Any(x => requestFormData.Any());
```

Comment:
Does this work?

Reply:
Yep it does, I debugged and checked if there is no values given then it fails and if there is at least one then it passes

Reply:
Even if there is irrelevant form data, just not one of the required ones?
E.g.
```
{
  "some-field": "test"
}
```
Should still fail because it doesn't contain one of: title, firstName, LastName, etc